includes/Utilities/BackgroundJobCleanup.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace SkyVerge\WooCommerce\Facebook\Utilities;
```

Comment:
Why not move this into `namespace SkyVerge\WooCommerce\Facebook\Jobs`? We don't want the Utilities directory to become a dumping ground.

Reply:
Originally it was because this was potentially targeting jobs from other places. It is a bit of a script/utility, I'll look at moving it.